front/src/modules/auth/components/Modal.tsx

```diff
@@ -1,22 +1,13 @@
 import React from 'react';
-import styled from '@emotion/styled';
 
 import { Modal as UIModal } from '@/ui/modal/components/Modal';
 
 type Props = React.ComponentProps<'div'>;
 
-const StyledContainer = styled.div`
```

Comment:
Keep the width property, otherwise, this is breaking our auth flow using AuthModal:

<img width="1512" alt="image" src="https://github.com/twentyhq/twenty/assets/12035771/02141168-8f59-4efc-b507-2414d4ed5b97">